app/views/membership_applications/_membership_application_fields.html.haml

```diff
@@ -78,7 +78,8 @@
     var upload_table = document.getElementById("files-to-upload");
 
     var filesUploadButton = document.getElementById("upload-button");
-    var txt = "";
+
+    $(upload_table).find("tr>td").remove();
```

Comment:
I think a user should be able to _add_ to the list of uploaded files.

But for now, if we don't want to handle that, we should at least **warn the user** that their current list of files will be wiped out.

Reply:
@weedySeaDragon - see below the suggestion to add some help text explaining how to upload multiple files.  I think that negates the need for a warning about wiping out previously-selected files (and adding both the help text for multiple selection _and_ the warning text could be very confusing, IMO).